### PR TITLE
Add Daum app user agent

### PR DESCRIPTION
**Pull request by HyewonKkang**
# Prerequisites

- [x] I have read and follow the [contributing](https://github.com/faisalman/ua-parser-js/blob/master/CONTRIBUTING.md) guidelines
- [x] I have read and accept the [Contributor License Agreement (CLA)](https://gist.github.com/faisalman/2ed16621ebb544157eba85a7f7381417) Document and I hereby sign the CLA

# Type of Change

enhancement

# Description

- Added support for recognizing the 'Daum' app in user agent strings.
- AS-IS: 'Daum' app was not identified.
- TO-BE: User agents with 'Daum' are now accurately parsed.

application download link: [google play](https://play.google.com/store/apps/details?id=net.daum.android.daum&hl=ko), [apple apps](https://apps.apple.com/us/app/%EB%8B%A4%EC%9D%8C-daum/id365494029)

# Test

- Added tests for 'Daum' user agent string.
- Ran the full test suite successfully.

# Impact

Breaking Changes: None
User Actions: No changes required, backward compatibility maintained

# Other Info

### CODE REVIEW

**Review by faisalman — CHANGES_REQUESTED** [review]
Hi @HyewonKkang, Thank you for contributing to this project! 

- Please commit only the source files and tests. You can omit the `.mjs` files as they will be autogenerated.
- The value in the `Browser` enum should match with the browser name in the result:

```js
import { UAParser } from 'ua-parser-js';
import { Browser } from 'ua-parser-js/enum';

const result = UAParser(navigator.userAgent);

// string comparison must be the same
if (result.browser.name == Browser.DAUM) {
  console.log('You are using Daum app');
}
```

**Comment by HyewonKkang** [comment]
@faisalman 
Thank you for reviewing quickly.
I've applied the review comments, so please check.

**Review by faisalman — CHANGES_REQUESTED** [review]
Hi, thanks for following up, but this issue still hasn't been addressed:

>     * The value in the `Browser` enum should match with the browser name in the result:
> 
> ```js
> // string comparison must be the same
> if (result.browser.name == Browser.DAUM) {
> ```

You can change the captured string into, for example:

```js
/(daum)apps[\/ ]([\w\.]+)/i,
```

So both of them will match

File: test/data/ua/browser/browser-all.json [hunk 1]
```diff
@@ -2503,6 +2503,26 @@
             "type"    : "inapp"
         }
     },
+    {
+      "desc"     : "Daum App Android",
+      "ua"       : "Mozilla/5.0 (Linux; Android 11; SM-G970N Build/RP1A.200720.012; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/83.0.4103.106 Mobile Safari/537.36 DaumApps/7.5.0 DaumDevice/mobile",
+      "expect"   : {
+        "name"   : "DaumApps",
```
Comment (faisalman) [inline test/data/ua/browser/browser-all.json:2510]: Don't forget to also change the expected name to `Daum`.

**Review by faisalman — CHANGES_REQUESTED** [review]
Change the value in the test files as well and run `npm test` to make sure that the change is all good.

File: test/data/ua/browser/browser-all.json (hunk 1 above)
Comment (HyewonKkang) [reply test/data/ua/browser/browser-all.json]: I missed it, thanks 👋🏻

**Review by faisalman — APPROVED** [review]
All seems good now, thanks!

**Pull request merged**